springfield/firefox/templates/firefox/enterprise/index.html

```diff
@@ -56,6 +56,20 @@ <h1 class="mzp-c-split-title">{{ftl('firefox-enterprise-get-firefox-for-your-ent
     </div>
   {% endcall %}
 
+  <section class="mzp-c-callout enterprise-callout">
+    <div class="mzp-l-content mzp-t-content-md callout-wrapper">
+      <div class="mzp-c-callout-content">
+        <h2 class="mzp-c-callout-title">{{ ftl('firefox-enterprise-professional-support') }}</h2>
+        <div class="mzp-c-callout-desc">
+          <p>{{ ftl('firefox-enterprise-whether-you-are') }}</p>
+        </div>
+        <a class="callout-cta mzp-c-button mzp-t-product" href="https://qsurvey.mozilla.com/s3/firefox-support-plan?src=firefox-enterprise-cta/{{ params }}" data-cta-text="Contact Sales">
```

Comment:
We already have some params as `?src=firefox-enterprise-cta`:

```suggestion
        <a class="callout-cta mzp-c-button mzp-t-product" href="https://qsurvey.mozilla.com/s3/firefox-support-plan?src=firefox-enterprise-cta" data-cta-text="Contact Sales">
```